Azure-Samples/service-fabric-cluster-templates
# Sf sample template

## Purpose

* ... The purpose of this ARM (Azure Resource Manager) template is to facilitate the seamless provisioning of a Service Fabric cluster while ensuring the security and protection of the associated storage account. By implementing the template, you can create a Service Fabric cluster that is shielded from public network access, thereby minimizing potential security risks.
One of the key features of this template is the ability to configure access to the storage account only from a whitelisted network/IP. This ensures that access to the storage account is restricted to specific networks or IP addresses, thereby enhancing the overall security posture of your Service Fabric cluster.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
As mentioned in the readme.md file attached in the folder.
Download the template file on your machine. 
Make sure you have an existing virtual network on which you will be deploying your cluster & whitelisting it in Storage account Firewall rules from ARM template.
Then make the required changes in the template file as mentioned in readme.md.
Lastly deploy the template to Azure.


```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
